platform/macos/src/MGLMapView.mm

```diff
@@ -102,7 +102,7 @@
 
 /// Mapping from an annotation tag to metadata about that annotation, including
 /// the annotation itself.
-typedef std::map<MGLAnnotationTag, MGLAnnotationContext> MGLAnnotationContextMap;
+typedef std::unordered_map<MGLAnnotationTag, MGLAnnotationContext> MGLAnnotationContextMap;
```

Comment:
👍 This is a good choice for what’s essentially a sparse array. By the way, the iOS SDK implementation of MGLMapView defines its own `MGLAnnotationContextMap` and uses it the same way as this implementation.